tests/test_api.py

```diff
@@ -265,30 +265,51 @@ def test_event_error_rate(truth, prediction, eer):
 ])
 def test_concordance_cc(truth, prediction):
 
+    def expected(truth, prediction):
+        if len(prediction) < 2:
+            ccc_expected = np.NaN
+        else:
+            denominator = (
+                prediction.std() ** 2
+                + truth.std() ** 2
+                + (prediction.mean() - truth.mean()) ** 2
+            )
+            if denominator == 0:
+                ccc_expected = np.NaN
+            else:
+                r = np.corrcoef(list(prediction), list(truth))[0][1]
+                ccc_expected = (
+                    2 * r * prediction.std() * truth.std()
+                    / denominator
+                )
+        return ccc_expected
+
     ccc = audmetric.concordance_cc(truth, prediction)
 
     prediction = np.array(list(prediction))
     truth = np.array(list(truth))
 
-    if len(prediction) < 2:
-        ccc_expected = np.NaN
-    else:
-        denominator = (
-            prediction.std() ** 2
-            + truth.std() ** 2
-            + (prediction.mean() - truth.mean()) ** 2
-        )
-        if denominator == 0:
-            ccc_expected = np.NaN
-        else:
-            r = np.corrcoef(list(prediction), list(truth))[0][1]
-            ccc_expected = 2 * r * prediction.std() * truth.std() / denominator
-
     np.testing.assert_almost_equal(
         ccc,
-        ccc_expected,
+        expected(truth, prediction),
     )
 
+    # Handle NaN in prediction
```

Comment:
```suggestion
    # Handle NaN in prediction and truth
```

Reply:
I reworked the tests and moved them to a separate file. The comment from above is no longer present.